packages/internals/package.json

```diff
@@ -88,7 +88,7 @@
     "@prisma/generator-helper": "workspace:*",
     "@prisma/get-platform": "workspace:*",
     "@prisma/prisma-schema-wasm": "6.6.0-52.30389eb8d83fe5c09a7d1d1648e7d5ab7438d1de",
-    "@prisma/schema-engine-wasm": "6.6.0-52.30389eb8d83fe5c09a7d1d1648e7d5ab7438d1de",
+    "@prisma/schema-engine-wasm": "6.6.0-53.integration-0d6517673fbba640b066c981502e5df705c2cf08",
```

Comment:
Note: this is temporary. I'm waiting for the actual engine version pushed by @prisma-bot.